src/SymbolServer.jl

```diff
@@ -27,6 +27,57 @@ mutable struct SymbolServerInstance
     end
 end
 
+function get_general_pkgs()
+    @static if VERSION >= v"1.7-"
+        regs = Pkg.Types.Context().registries
+        i = findfirst(r -> r.name == "General" && r.uuid == UUID("23338594-aafe-5451-b93e-139f81909106"), regs)
+        i === nothing && return Dict{UUID, PkgEntry}()
+        return regs[i].pkgs
```

Comment:
This isn't working in the vscode extension because `Pkg.Types.Context().registries` is empty
```
┌ Info: regs
└   regs = Pkg.Registry.RegistryInstance[]
┌ Error: Symbol cache downloading: Failed to identify which packages to omit based on the General registry.
│ All packages will be processsed locally
│   err = Could not find the General registry
└ @ SymbolServer ~/Documents/GitHub/julia-vscode/scripts/packages/SymbolServer/src/
```

Reply:
What's the right way to get the General registry from here?

Reply:
That's kinda tricky. We're overwriting setting the `JULIA_DEPOT_PATH` env variable to a depot vendored in the extension, but you could probably temporarily change the DEPOT_PATH before this?

Reply:
Ok. I've tried to implement that. I'd appreciate trying to get this merged to stop sending out non-General package name & uuids